requirements.txt

```diff
@@ -23,3 +23,5 @@ matplotlib
 wandb
 python-dotenv
 pytest
+torch_sparse
```

Comment:
torch_sparse and torch-scatter is already specified in line 18 and 19. So we can close this PR without merging.